contrib/rcirc/packages.el

```diff
@@ -13,23 +13,12 @@ which require an initialization must be listed explicitly in the list.")
 (defun rcirc/init-rcirc ()
   (use-package rcirc
     :defer t
+    :commands rcirc
```

Comment:
not necessary, if you `SPC m g g` on `rcirc` you can see that it has an `;;;###autoload` cookie which mean the `:defer t` is sufficient.